types/node/index.d.ts

```diff
@@ -2017,7 +2004,7 @@ declare module "url" {
         hash?: string;
     }
 
-    export function parse(urlStr: string, parseQueryString?: boolean, slashesDenoteHost?: boolean): Url;
+    export function parse(urlStr: string, parseQueryString?: boolean, slashesDenoteHost?: boolean): URL;
```

Comment:
I think this is not correct. NodeJS offers two Url apis:
- the legacy api (https://nodejs.org/dist/latest-v8.x/docs/api/url.html#url_legacy_url_api) matching to the just removed Url interface.
- the WHATWG URL API (https://nodejs.org/dist/latest-v8.x/docs/api/url.html#url_the_whatwg_url_api) using `class URL`.

`url.parse()` returns a plain JS Object matching interface Url but no class URL .